pyproject.toml

```diff
@@ -32,6 +32,8 @@ lxml = "^4.6.2"
 geopy = "^2.1.0"
 tzwhere = "^3.0.3"
 backoff = "^1.11.1"
+chardet = "^5"
+types-chardet = "^5.0.4"
```

Comment:
We should move this to `tool.poetry.dev-dependencies`, because it's only needed for typing analysis

Reply:
This have been updated in the latest push

Also tested locally and got the following results
```
LOCAL - Running command black --check --diff .
All done! ✨ 🍰 ✨
41 files would be left unchanged.
LOCAL - Running command flake8 .
LOCAL - Running command find . -name "*.py" | xargs pylint
************* Module tasks
tasks.py:4:0: W0402: Uses of a deprecated module 'distutils.util' (deprecated-module)

------------------------------------------------------------------
Your code has been rated at 9.99/10 (previous run: 9.99/10, +0.00)
```